FWCore/Framework/interface/EventSetupRecord.h

```diff
@@ -258,6 +244,26 @@ namespace edm {
       bool requireTokens() const { return requireTokens_; }
 
     private:
+      template <typename HolderT>
```

Comment:
This is just the old `get` call renamed. I did this to avoid having multiple "deprecated" messages happening in the case someone called a deprecated function that then called the deprecated `get`. In that case you'd get one warning at the callers site and a second one in `EventSetupRecord.h` itself. Moving the implementation to a private function that isn't annotated avoids that.